qaframework-bdd-tests/cypress/support/commands.js

```diff
@@ -44,7 +44,10 @@ Cypress.Commands.add('login', () => {
     cy.request({
         method: 'POST',
         url: `${API_BASE_URL}/session`,
-        body: {sessionLocation: DEFAULT_LOCATION_UUID},
+        body: {
+          sessionLocation: DEFAULT_LOCATION_UUID, 
+          locale: "en_GB",
```

Comment:
this solves  500 server error `'null' is not in the list of allowed locales` thrown with `cy.login()`
i will go ahead and create a separate PR for it latter on